rest/src/routes/networkRoutes.js

```diff
@@ -69,10 +71,16 @@ module.exports = {
 		server.get('/network/fees/transaction', (req, res, next) => {
 			const numBlocksTransactionFeeStats = services.config.numBlocksTransactionFeeStats || 1;
 			const latestBlocksFeeMultiplier = db.latestBlocksFeeMultiplier(numBlocksTransactionFeeStats);
-			return Promise.all([readAndParseNodePropertiesFile(), latestBlocksFeeMultiplier]).then(feeMultipliers => {
```

Comment:
Configuration files should be loaded and sanitized once, not every request. Each request could just use the loaded object. 
This is a refactoring that's better to have types, so I would refactor/reimplement this once we have Typescript